lib/usd/translators/meshReader.cpp

```diff
@@ -113,18 +113,11 @@ MayaUsdPrimReaderMesh::Read(UsdMayaPrimReaderContext* context)
                                          &status);
     CHECK_MSTATUS_AND_RETURN(status, false);
 
-    // shape name
-    const auto& primName = prim.GetName().GetString();
-    status = UsdMayaUtil::setName(meshRead.meshObject(), primName);
-    CHECK_MSTATUS_AND_RETURN(status, false);
```

Comment:
This was just wrong! UsdMayaUtil::setName was creating a new object of type MFnMesh and set it's name.